Mathlib/Data/List/MinMax.lean

```diff
@@ -449,6 +449,11 @@ theorem minimum_of_length_pos_le_iff (h : 0 < l.length) :
     minimum_of_length_pos h ≤ b ↔ l.minimum ≤ b :=
   le_maximum_of_length_pos_iff (α := αᵒᵈ) h
 
+theorem minimum_of_length_pos_mem (h : 0 < l.length) :
```

Comment:
Can you add the `maximum` version too?

Reply:
Yes! I have added the `maximum` version.

Reply:
Thanks! Can you swap the order to match the surrounding file? Elsewhere it seems the maximum version of the lemma comes before the minimum

Reply:
Yes! it is done.